docs/detections/rules-ui-manage.asciidoc

```diff
@@ -117,7 +122,16 @@ You can duplicate, enable, disable, and delete rules:
 [[import-export-rules-ui]]
 === Export and import rules
 
-You can export detection rules to an `.ndjson` file, which you can then import into another {elastic-sec} environment. The following configuration items are also included in the `.ndjson` file:
+You can export custom detection rules to an `.ndjson` file, which you can then import into another {elastic-sec} environment. 
+
+[NOTE]
+====
+You cannot export Elastic prebuilt rules, but you can duplicate a prebuilt rule and then export the duplicated rule.
```

Comment:
```suggestion
You cannot export Elastic prebuilt rules, but you can duplicate a prebuilt rule, then export it.
```

Reply:
I think we still need to reiterate "the duplicated rule" at the end for clarity, since grammatically "it" points back to "a prebuilt rule" since it's the closest noun.
```suggestion
You cannot export Elastic prebuilt rules, but you can duplicate a prebuilt rule, then export the duplicated rule.
```